docs/docs/docs/01-core/admin/04-hooks/use-enterprise.mdx

```diff
@@ -54,3 +54,8 @@ if (!Component) {
     return;
 }
 ```
+
+### `enabled`
+
+Similar to react-query this boolean flag allows disabling the EE import, e.g. when more than one condition needs to be applied. If `enabled`
+is set to false, the first argument (CE_DATA) will be returned.
```

Comment:
would a good example of this be where the EE addition is unique to a feature being enabled as opposed to EE in general?